test/bluesky.auth.xrpc.test.ts

```
@@ -0,0 +1,111 @@
+    expect(headersSeen[0]!.dpop.length).toBeGreaterThan(20);
+  });
+
+  it('refreshes on 401 then retries XRPC', async () => {
+    const session = await baseSession();
+    let xrpc = 0;
+    vi.spyOn(globalThis, 'fetch').mockImplementation(async input => {
+      const url = String(input);
+      if (url.includes('/xrpc/app.bsky.notification.getUnreadCount')) {
+        xrpc += 1;
+        if (xrpc === 1) {
+          return new Response(JSON.stringify({ error: 'ExpiredToken', message: 'exp' }), { status: 401 });
```

ESLint

[error] Replace `·status:·401` with `⏎············status:·401⏎·········`

(prettier/prettier)